wolox-books-angularjs/src/app/components/centered/book-detail/book-detail.pug

```diff
@@ -12,7 +12,7 @@ a.return(ui-sref='centered.dashboard')
       span.book-item {{bookDetailCtrl.book.year}}
       span.book-item {{bookDetailCtrl.book.genre}}
       span.book-desc Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
-      button.rent-btn {{'BookDetail.rent' | translate}}
+      button.blue-btn {{'BookDetail.rent' | translate}}
```

Comment:
why not `button.btn.blue`

Reply:
buttons differ in other styles than color